README.md

```diff
@@ -1,2 +1,77 @@
-# dpel-website
-Website for the Digital Production Example Library
+# DPEL Website Doc
+
+**DPEL: Digital Production Example Library**
+A library of digital assets - 3D scenes, digital cinema footage, etc. - that demonstrate the scale and complexity of modern feature film production, including computer graphics, visual effects and animation.
+
+## 🚀 Project Structure
+
+Inside the DPEL project, you'll see the following folders and files:
+
+```
+/
+├── public/
+│   └── favicon.svg ( This is the icon that shows up in the browser tab )
```

Comment:
It looks like this file is currently the Astro icon.  Should this be updated with the ASWF DPEL icon?

Reply:
I'll update that!